src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/RuntimeInformation.cs

```diff
@@ -70,6 +70,8 @@ public static Architecture ProcessArchitecture
             => Architecture.LoongArch64;
 #elif TARGET_POWERPC64
             => Architecture.Ppc64le;
+#elif TARGET_RISCV64
+            => Architecture.LoongArch64; // TODO-RISCV64: go though API review for RiscV64
```

Comment:
instead of reusing `LoongArch64` you can do `(Architecture)9` (which is the expected next enum value if we add a RiscV64 entry)

Reply:
Makes sense.
```suggestion
            => (Architecture)9; // TODO-RISCV64: go though API review for RiscV64
```